packages/lesswrong/components/seasonal/HomepageCommunityMap.tsx

```diff
@@ -1,38 +1,1204 @@
+    "lng": 16.9351199
+  },
+  {
+    "_id": "gYtS4Z3Deu8T4mWx4",
+    "lat": 43.234353,
+    "lng": -77.539637
+  },
+  {
+    "_id": "3sf5BvRaDvFFuR7fw",
+    "lat": 30.3058125,
+    "lng": -97.7389375
+  },
+  {
+    "_id": "SgPWHxawrek2PCYaG",
+    "lat": 43.5884824,
+    "lng": 1.4463867
+  },
+  {
+    "_id": "WLXEnXHwBzynC9daj",
+    "lat": 38.62700249999999,
+    "lng": -90.1994042
+  },
+  {
+    "_id": "uKqND22xB2FvA3ejw",
+    "lat": 38.6074337,
+    "lng": -90.2578509
+  },
+  {
+    "_id": "apr797RGL9RgzhKpM",
+    "lat": 40.7467891,
+    "lng": -111.8763126
+  },
+  {
+    "_id": "zYkY8dKMAwTCYorTG",
+    "lat": 33.9963606,
+    "lng": -118.4156462
+  },
+  {
+    "_id": "fdPdvbnXAnhjoMEwC",
+    "lat": 19.1328771,
+    "lng": 72.81231989999999
+  },
+  {
+    "_id": "xqMXSYJMLaLf9YnC8",
+    "lat": 35.6541917,
+    "lng": -97.4778707
+  },
+  {
+    "_id": "iferav2xZHi6dfaZT",
+    "lat": 32.7392362,
+    "lng": -117.133824
+  },
+  {
+    "_id": "Bcyko7BskBLbPHTCt",
+    "lat": 47.6128,
+    "lng": -122.3205327
+  },
+  {
+    "_id": "zZApZEyKqHCprHNPC",
+    "lat": 30.37128869999999,
+    "lng": -87.1769198
+  },
+  {
+    "_id": "bhAwA6CeQAwyz2BPp",
+    "lat": 53.5461245,
+    "lng": -113.4938229
+  },
+  {
+    "_id": "4uaEuBkaKQDbuyvgh",
+    "lat": 47.9981645,
+    "lng": 7.8576265
+  },
+  {
+    "_id": "mo7LkrMAxnh5S3ksr",
+    "lat": 28.6024274,
+    "lng": -81.2000599
+  },
+  {
+    "_id": "RkiEmLe7s5R2tdAn8",
+    "lat": 54.00973759999999,
+    "lng": -2.7859802
+  },
+  {
+    "_id": "qfktAPKrDRiwMv9Am",
+    "lat": 33.960048,
+    "lng": -83.382717
+  },
+  {
+    "_id": "kbQGxGXhckpnstkG8",
+    "lat": 33.4930598,
+    "lng": -112.1113979
+  },
+  {
+    "_id": "sRGECzX6fnfFchrbR",
+    "lat": 44.4289447,
+    "lng": 26.0351844
+  },
+  {
+    "_id": "9JpZ4XHvkrnt6NbBh",
+    "lat": 39.147744,
+    "lng": -86.537783
+  },
+  {
+    "_id": "BrsnfyKFDAzwptJHg",
+    "lat": 40.01907629999999,
+    "lng": -105.1379106
+  },
+  {
+    "_id": "pMjEs8y9CMjHCmKXA",
+    "lat": 31.7493125,
+    "lng": 35.18681249999999
+  },
+  {
+    "_id": "DKMphwNAgcCwgZABQ",
+    "lat": 51.7493921,
+    "lng": -1.2383467
+  },
+  {
+    "_id": "jyRkrCkhwYPzEopH5",
+    "lat": 40.8908622,
+    "lng": -73.3540881
+  },
+  {
+    "_id": "Tjrw8JzgaW9zahZM2",
+    "lat": 43.7194164,
+    "lng": 10.4000338
+  },
+  {
+    "_id": "KERimTihvPXu3bKB7",
+    "lat": 28.6024274,
+    "lng": -81.2000599
+  },
+  {
+    "_id": "cavXhfgn3CrmEpHhn",
+    "lat": 28.6024274,
+    "lng": -81.2000599
   }
+]
+
+const styles = communityMapStyles
+
+const LocalEventWrapperPopUp = ({localEvent, handleClose}:{
+  localEvent:LocalEvent,
+  handleClose: (eventId: string)=>void
+}) => {
+  const { StyledMapPopup, GroupLinks } = Components
+  const { document, loading } = useSingle({
+    documentId: localEvent._id,
+    collectionName: "Posts",
+    fragmentName: 'PostsList',
+  });
+
+  if (!localEvent.lat || !localEvent.lng) return null
```

Comment:
(also the type `LocalEvent` doesn't permit either of those fields to be null or undefined, so the only time which this would return `null` would be if either of them were the value `0`)